doc/source/user_guide/scale.rst

```diff
@@ -374,5 +374,33 @@ datasets.
 
 You see more dask examples at https://examples.dask.org.
 
+Use Modin
```

Comment:
Just my two cents. I would just simply remove both Dask and Modin here, add a link to https://github.com/pandas-dev/pandas/blob/a0784d2c59c04e0a9e1ef22d696adb5768b5541e/web/pandas/community/ecosystem.md?plain=1#L344

A lot of information here is duplicated with Dask vs Modin and others too such as Pandas API on Spark (Koalas). They all aim drop-in replacement, they all out-of-core in multiple nodes, partitioning, scalability.

Reply:
Or, I would at least deduplicate them here if we want to keep those sections. e.g.,

```
# Using thirdparty libraries

multiple nodes blah blah..

## Dask

oneliner explanation

Link

## Modin

oneliner explanation

Link

## PySpark: Pandas API on Spark

oneliner explanation

Link


```

Reply:
> A lot of information here is duplicated with Dask vs Modin and others too such as Pandas API on Spark (Koalas). They all aim drop-in replacement, they all out-of-core in multiple nodes, partitioning, scalability.

I would argue against the statement `They all aim drop-in replacement`. Modin is supposed to work after changing the import statement `import pandas as pd` to `import modin.pandas as pd` and vice versa, while Dask or Koalas require the code to be rewritten to the best of my knowledge. Anyway, if everyone thinks that `Scaling to large datasets` page should only contain pandas related stuff, I can go ahead and remove Modin and Dask related stuff from that page and move the changes from this PR to the ecosystem page. Thoughts?

Reply:
I'm not really a maintainer here and don't have a super strong opinion. I'll defer to pandas maintainers.